source/enterprise/tech-guide/backup-eca-guide.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: Backup the Client Administration
+github_link: enterprise/tech-guide/backup-eca-guide.md
+indexed: false
+---
+
+This guide describes how you backup the Client Administration.
+
+<div class="toc-list"></div>
+
+## How to create a Backup
+
+### 1. The mysql database.
+
+First of all, you have to back up the mysql database.
+
+The database is called __shopware_client_administration__.
```

Comment:
not every sentence needs its own row. Sentences are like family, usually they stick together.